rm/ut/rm_service.c

```diff
@@ -44,14 +43,12 @@ static char *server_argv[] = {
 
 extern struct m0_reqh_service_type m0_rms_type;
 
-static struct m0_net_xprt *xprt        = &m0_net_lnet_xprt;
 static struct rm_ctx  *server_ctx  = &rm_ctxs[SERVER_1];
 static struct rm_ctx  *client_ctx  = &rm_ctxs[SERVER_2];
 static struct m0_clink     tests_clink[TEST_NR];
 extern void flock_client_utdata_ops_set(struct rm_ut_data *data);
 
 static struct m0_rpc_server_ctx sctx = {
-	.rsx_xprts            = &xprt,
 	.rsx_xprts_nr         = 1,
```

Comment:
This `nr` is not changed, when we assign the sctx.rsx_xprts at LINE 74?

Reply:
incorporated